### PR TITLE
Update customizing-gitea.en-us.md

### PR DESCRIPTION
**Pull request by yusefmaali**
Some changes to the update page.

### CODE REVIEW

**Comment by yusefmaali** [comment]
Ok, this time I've done the pull request in the correct repository...

File: content/doc/advanced/customizing-gitea.en-us.md [hunk 1]
```diff
@@ -17,14 +17,32 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+If you installed Gitea from binary, you can find the `custom` folder next to the binary.
+You should already find it, with a `conf` folder inside, where Gitea stores all the configuration settings provided through the installation steps.
+
```
Comment (strk) [inline content/doc/advanced/customizing-gitea.en-us.md]: Which binary contains the `custom` folder ? Last time I checked he official binary downloads were a single executable file...

**Review by strk — CHANGES_REQUESTED** [review]
Looks good to me, except for the "install from binary" observation

**Comment by yusefmaali** [comment]
committed the fixes.
the sense should now be more clear

**Review by strk — COMMENTED** [review]
Should you mention somewher ethat the `custom` folder can be configured to be elsewhere via an environment variable ?

**Comment by yusefmaali** [comment]
Can you give me an entry point to read something about that? Should be nice to add a short reference about this feature.

**Comment by strk** [comment]
See #111 (which suggests the info should already be in the file)

**Comment by yusefmaali** [comment]
done

**Comment by strk** [comment]
thanks, LGTM

**Comment by andreynering** [comment]
LGTM, sorry for the delay

**Pull request merged**